Description: build: Fix doc install path
 Missing mkdir breaks building docs only
Forwarded: yes
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Last-Update: 2016-01-18

diff --git a/mk/rte.sdkinstall.mk b/mk/rte.sdkinstall.mk
index c159bf7..68e56b6 100644
--- a/mk/rte.sdkinstall.mk
+++ b/mk/rte.sdkinstall.mk
@@ -157,4 +157,5 @@ ifneq ($(wildcard $O/doc/*/*/*pdf),)
 	$(Q)$(call rte_mkdir,     $(DESTDIR)$(docdir)/guides)
 	$(Q)cp -a $O/doc/*/*/*pdf $(DESTDIR)$(docdir)/guides
 endif
+	$(Q)$(call rte_mkdir,         $(DESTDIR)$(datadir))
 	$(Q)cp -a $(RTE_SDK)/examples $(DESTDIR)$(datadir)
